playground/frontend/lib/modules/editor/repository/code_repository/code_client/grpc_code_client.dart

```diff
@@ -83,13 +83,14 @@ class GrpcCodeClient implements CodeClient {
   ) {
     return _runSafely(() => createClient(request.sdk)
         .getRunOutput(grpc.GetRunOutputRequest(pipelineUuid: pipelineUuid))
-        .then((response) => _toOutputResponse(response.output)));
+        .then((response) => _toOutputResponse(response.output))
+        .catchError((err) => _toOutputResponse('')));
```

Comment:
There should be a logger in the `catchError` so that we can find potential problems.